src/test/java/org/fundacionjala/coding/ariel/Kata3Test.java

```diff
@@ -0,0 +1,37 @@
+package org.fundacionjala.coding.ariel;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * It test check five conditions..
+ */
+public class Kata3Test {
+
+    public static final int R1 = 3;
+    public static final int R2 = 0;
+    public static final int R3 = 2;
+    public static final int R4 = 4;
+    public static final int R5 = 3;
+    public static final int INPUT1 = 39;
+    public static final int INPUT2 = 4;
+    public static final int INPUT3 = 25;
+    public static final int INPUT4 = 999;
+    public static final int INPUT5 = 444;
+
+    /**
+     * It test made the five conditions.
+     */
+    @Test
+    public void basicTests() {
+        System.out.println("****** Basic Tests ******");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)